src/bundlers/utils.ts

```diff
@@ -87,3 +87,80 @@ export async function bundle(
     debugLogger.debug(`The bundling process finished successfully for file ${element.path}.`);
     return output;
 }
+
+export function getLocalFunctionWrapperCode(handler: string, entry: string, language: string) {
```

Comment:
[nit] We can be more strict with the `language` here:

```suggestion
export function getLocalFunctionWrapperCode(handler: string, entry: string, language: Language) {
```